### PR TITLE
add kegmutator and kegType props

### PR DESCRIPTION
**Pull request by AntonPuko**

### CODE REVIEW

File: src/index.js.flow [hunk 1]
```diff
@@ -225,6 +238,13 @@ export type Keg = {
   tapDate: Date,
 };
 
+export type KegMutator = {
```
Comment (AntonPuko) [inline src/index.js.flow]: you said that its probably should be also flow sensor id, but idk
there isn't that prop here: https://github.com/Brewskey/Brewskey.Web/blob/master/Source/Keg.Web/Models/Kegs/EditKeg.cs

I was trying 'flowSensorId'  -doesn't' work. and why do we need flowSensorId for setup keg at all?

File: src/dao/KegDAO.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+import type { Keg, KegMutator, KegType, KegTypeProps } from '../index';
 
 import DAO from './DAO';
 import KegTranslator from '../translators/KegTranslator';
 import LoadObject from '../LoadObject';
 import { DAO_ENTITIES } from '../constants';
 import { createFilter } from '../filters';
 
-class KegDAO extends DAO<Keg, Keg> {
+export const KEG_PROPS_BY_TYPE: { [KegType]: KegTypeProps } = {
```
Comment (jlkalberer) [inline src/dao/KegDAO.js]: I don't know if the name belongs here. It seems more like a presentation detail.

I'm thinking that you should probably store the enum value here.

`size` should probably be `maximumOunces`

File: src/index.js.flow (hunk 1 above)
Comment (jlkalberer) [reply src/index.js.flow]: Yeah... I'll add the mutator class to the controller and that should fix things. I don't think the controller will work right now as is.

File: src/dao/KegDAO.js (hunk 2 above)
Comment (AntonPuko) [reply src/dao/KegDAO.js]: yeah,`name` is presentation, but maximOunces looks more like data?

File: src/index.js.flow [hunk 3]
```diff
@@ -210,12 +210,25 @@ export type Glass = {
...
 
+export type KegType =
+  | 'Cornelius'
+  | 'HalfBarrel'
+  | 'Mini'
+  | 'QuarterBarrel'
+  | 'SixthBarrel'
+  | 'SlimQuarter';
+
+export type KegTypeProps = {
```
Comment (jlkalberer) [inline src/index.js.flow]: Is this used anywhere?

**Pull request merged**